Gopkg.toml

```diff
@@ -14,11 +14,17 @@
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  branch = "release-1.10"
+  version = "kubernetes-1.10.1"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "7.0.0"
+  version = "kubernetes-1.10.1"
+
+[[constraint]]
+  name = "sigs.k8s.io/controller-runtime"
+  source = "github.com/shawn-hurley/controller-runtime"
+  branch = "unstructured"
+#  version = "v0.1.1"
```

Comment:
looks like a stray that can be removed

Reply:
This should stay I think. We will eventually need to pin to a version.

Reply:
Cool. How about adding a quick comment to explain why it's here?